ansible_base/lib/utils/models.py

```diff
@@ -15,13 +15,13 @@
 logger = logging.getLogger('ansible_base.lib.utils.models')
 
 
-def get_all_field_names(model, concrete_only=False):
+def get_all_field_names(model, concrete_only=False, include_attnames=True):
```

Comment:
It would be nice to document this better. Like, a specific type hint about the return value, and a docstring that tells the function of all the method kwargs.